api/modules/github/views.py

```diff
@@ -45,3 +45,43 @@ def get_contributors(request, project):
         return Response(str(e), status=status.HTTP_503_SERVICE_UNAVAILABLE)
 
     return Response(response)
+
+
+@api_view(['GET'])
+def get_issues(request, project):
+    """
+        Return list of issues
+        :param request:
+        :param project:
+        :return: 503 if github api fails
+        :return: 200 successful
+        """
+    try:
+        api_response = requests.get(GITHUB_API_GET_ISSUES_URL.format(project_name=project))
```

Comment:
If I give an incorrect project name, I'm getting `503` with error message `"string indices must be integers"`
Add a better error message with `404` status code